app/src/main/java/com/zeapo/pwdstore/autofill/v2/ui/holder/PasswordViewHolder.kt

```diff
@@ -0,0 +1,13 @@
+package com.zeapo.pwdstore.autofill.v2.ui.holder
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.zeapo.pwdstore.R
+
+class PasswordViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    val type_image: ImageView = itemView.findViewById(R.id.type_image)
```

Comment:
```suggestion
    val typeImage: ImageView = itemView.findViewById(R.id.type_image)
```

No snake casing here please

Reply:
Mybad. Fixed